intermine/im-tables
# Missing url fix 170

Fixes issue with URLs missing from im-tables ( https://github.com/intermine/intermine/issues/2045 and https://github.com/intermine/im-tables/issues/170 )

This is dependent on https://github.com/intermine/imjs/pull/59 for the cors fix. 

To test: 
1. reproduce the error - You can reproduce the broken state by going to flymine and running this query:

```xml
<query model="genomic" view="DataSource.name DataSource.url" sortOrder="DataSource.name ASC" >
</query>
```

2. Generate html page code from flymine and save it.
3. Pull the PR changes, run `npm run dev` (assuming everything is npm installed already), and go to localhost:9000. 
4. Save the generated code somewhere the server can see it, e.g. as index.html in the root of im-tables. 
5. Make sure to change the url of im-tables to point to `dist/im-tables` and not the cdn version. 

To confirm the bug is fixed: urls should be present in the url table.
